test/compilable/test17908b.d

```diff
@@ -0,0 +1,10 @@
+// PERMUTE_ARGS:
+
+void foo(int);
+@disable void foo() {}
+alias g = foo;
+
+void main()
+{
+    g(10);
+}
```

Comment:
I don't see any use in this test case (also it's wrong ATM).

Reply:
I wanted to make sure that changing the order of the declarations will not influence the result (in the original bug report, one workaround in the code would have been to change the order of the function declarations.)

Reply:
@RazvanN7 it's simple to just put it in the same module as the other test - use a different name than `foo`.

Reply:
Yeah putting them all in one file when possible is the recommended approach (more files == slower autotester).
If you test for the order of declaration, then it makes sense. Could you add a comment before the second test case when you move them together, so it's more obvious to the clueless reader ?
Also, any specific reasons some don't have a body ?

Reply:
>Could you add a comment before the second test case when you move them together, so it's more obvious to the clueless reader ?

Sure thing.

>Also, any specific reasons some don't have a body ?
No, it was just a slip on my part.